modules/video/video.asset

```diff
@@ -0,0 +1,30 @@
+local layer = {
+  Identifier = "TestVideo",
+  Video = "C:/Users/ylvaselling/Documents/Work/Testmovies/result.mp4",
```

Comment:
Same as above;  if we need another test video, you can send it to me;  otherwise we can reuse one of the other test videos

Reply:
File should be moved to data/assets/examples either way if you want to keep it